src/pyhf/events.py

```diff
@@ -64,12 +64,18 @@ def _flush(self):
         self._callbacks = _callbacks
 
     def __call__(self, *args, **kwargs):
-        for func, arg in self.callbacks:
+        for func, arg in self._callbacks:
```

Comment:
@wernerd-cern Can you comment on why `self._callbacks` is necessary here? It seems like a different situation than with `_flush(self)` just above this function.